bundles/org.openhab.binding.liquidcheck/src/main/java/org/openhab/binding/liquidcheck/internal/httpclient/LiquidCheckHttpClient.java

```diff
@@ -62,7 +62,8 @@ public LiquidCheckHttpClient(LiquidCheckConfiguration config, HttpClient client)
     public String pollData() throws InterruptedException, TimeoutException, ExecutionException {
         String uri = "http://" + config.hostname + "/infos.json";
         Request request = client.newRequest(uri).method(HttpMethod.GET)
-                .timeout(config.connectionTimeout, TimeUnit.SECONDS).followRedirects(false);
+                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS).timeout(config.connectionTimeout, TimeUnit.SECONDS)
```

Comment:
A timeout was already defined here, you now defined a doubled timeouts.

Reply:
You could keep the original timeout here

Reply:
reverted to the original